test/file_collection_tests.coffee

```diff
@@ -301,10 +301,14 @@ if Meteor.isServer
               test.fail(err) if err
               testColl.importFile testfile, {}, bind_env (err, doc) ->
                 test.fail(err) if err
-                readstream = testColl.findOneStream {_id: doc._id }
-                readstream.on 'data', bind_env (chunk) ->
-                  test.equal chunk.toString(), '1234567890','Incorrect data read back from file stream'
-                readstream.on 'end', bind_env () ->
+                test.equal typeof doc, 'object', "No document imported"
+                if typeof doc == 'object'
+                  readstream = testColl.findOneStream {_id: doc._id }
+                  readstream.on 'data', bind_env (chunk) ->
+                    test.equal chunk.toString(), '1234567890','Incorrect data read back from file stream'
+                  readstream.on 'end', bind_env () ->
+                    onComplete()
+                else
                   onComplete()
```

Comment:
The issue was that `C:\tmp` didn't exist.  Once I created it, the test passed.  Ah, I think the thing to do is use `os.tmpdir()` instead of `/tmp`.  On Windows, this is `C:\Users\<myusername>\AppData\Local\Temp`.  Shall I make the change?